ip-messaging/rest/credentials/delete-credentials/delete-credentials.3.x.js

```diff
@@ -4,8 +4,10 @@ var Twilio = require('twilio').Twilio;
 
 var client = new Twilio(accountSid, authToken);
 
-client.chat.credentials('CRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX').delete().then(function(response) {
+client.chat.credentials('CRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
+  .remove()
+  .then(function(response) {
     console.log(response);
-}).catch(function(error) {
-    console.log(error);
-});
+  }).catch(function(error) {
+      console.log(error);
+  });
```

Comment:
As discussed, an incorrect use of the library, like for example calling the wrong function, will be shown in the console as a failed snippet. Bad parameters for example won't show in the console with or without `catch`.